src/OSPSuite.Core/Domain/Data/DataInfo.cs

```diff
@@ -29,7 +29,12 @@ public enum ColumnOrigins
       /// <summary>
       ///    Column is based on calculation values but is used in a different context (for example ParameterIdentification plots)
       /// </summary>
-      CalculationAuxiliary
+      CalculationAuxiliary,
+
+      /// <summary>
+      ///    Column is an identity curve or deviation line to facilitate the viewing of a plot 
+      /// </summary>
+      IdentityCurve
```

Comment:
Should we create two entries or call them all DeviationLine (identity line is just a special DeviationLine)

Reply:
we ll call them all deviation lines